presidio-anonymizer/tests/test_analyzer_result.py

```diff
@@ -239,8 +238,27 @@ def test_given_endpoint_larger_then_start_point_then_we_fail():
     assert e.value.err_msg == "Invalid input, analyzer result 10 must be smaller then 0"
 
 
-def test_given_endpoint_same_as_start_point_then_we_pass():
-    assert create_analayzer_result("", 0, 0, 0)
+def test_given_endpoint_equal_to_start_point_then_we_fail():
+    with pytest.raises(InvalidParamException) as e:
+        create_analayzer_result("", 0, 0, 0)
+    assert e.value.err_msg == "Invalid input, analyzer result 0 must be smaller then 0"
```

Comment:
That's not that clear.. what is analyzer result 0? maybe add 'start' and 'end'

Reply:
start and end are 0 it is possible someone gives you that.
since start 0 and end 1 is a valid point... we are checking 0,0 is invalid.

Reply:
I meant the text, 
maybe: Invalid input, analyzer result **start index** 0 must be smaller then **end index** 0
analyzer result 0 sounds like the first analyzer result in the list

Reply:
ohhh ok